game-core/src/test/java/games/strategy/triplea/settings/ProtectedStringClientSettingTest.java

```diff
@@ -0,0 +1,51 @@
+package games.strategy.triplea.settings;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+import org.sonatype.goodies.prefs.memory.MemoryPreferences;
+
+import games.strategy.security.CredentialManager;
+
+final class ProtectedStringClientSettingTest {
+  private final ProtectedStringClientSetting clientSetting = new ProtectedStringClientSetting("name", false);
+
+  @BeforeAll
+  static void setup() {
+    ClientSetting.setPreferences(new MemoryPreferences());
```

Comment:
You can get rid of this boilerplate by having the top-level fixture extend `AbstractClientSettingTestCase`.  Or you could put `testDisplayValue()` in its own nested fixture and have just that one fixture extend `AbstractClientSettingTestCase`.